x/src/test.rs

```diff
@@ -19,13 +24,36 @@ pub struct Args {
     unit: bool,
     #[structopt(name = "TESTNAME", parse(from_os_str))]
     testname: Option<OsString>,
+    #[structopt(name = "LCOVFILE", parse(from_os_str))]
+    lcovfile: Option<OsString>,
+    #[structopt(name = "HTMLCOVDIR", parse(from_os_str))]
+    htmlcovdir: Option<OsString>,
```

Comment:
Maybe it would be worth having a new command for running coverage reports?

Reply:
done